pt-lua.c

```diff
@@ -983,6 +983,10 @@ int main (int argc, char **argv) {
   /* -------- PALLENE TRACER CODE -------- */
   (void) pallene_tracer_init(L);  /* initialize pallene tracer */
   lua_pop(L, 1);  /* We do not need the finalizer object here */
+
+  /* supply the message handler function with custom tracebacks. */
+  lua_pushcfunction(L, msghandler);
+  lua_setglobal(L, "pallene_tracer_errhandler");
```

Comment:
Nitpick: can we call this a "message handler" instead of "errhandler", to fit with the names that xpcall uses.

Reply:
Actually this message handler is the error handler here. But, error handler in this case would be much more intuitive.